schema/crdb/23.0.0/up4.sql

```diff
@@ -36,6 +36,7 @@ CROSS JOIN omicron.public.silo AS s
 WHERE p.time_deleted IS null 
   AND p.silo_id IS null -- means it's a fleet pool
   AND s.time_deleted IS null
+  -- TODO: AND NOT s.discoverable ???
```

Comment:
Leaning toward not adding this additional filter. The goal is to convert existing data to the new model while keeping the logic the same, and if P0 is a fleet default pool before this migration, it is accessible from non-discoverable silos. So in order to keep the behavior the same _after_ the migration, P0 should be linked to those silos.